.github/workflows/extra-docs-linting.yml

```
@@ -0,0 +1,32 @@
+name: Lint extra docsite docs and links
+on:
+  # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
+  push:
+    branches:
+      - main
+      - stable-*
+  pull_request:
+  # Run CI once per day (at 06:00 UTC)
+  # This ensures that even if there haven't been commits that we are still testing against latest version of ansible-test for each ansible-base version
+  schedule:
+    - cron: '0 6 * * *'
+
+jobs:
+  docsite:
+    name: Lint extra docsite docs and links
+    runs-on: ubuntu-latest
```

This isn't strictly necessary but I think we should start specifying permissions as a good practice, to scope down the token to least privilege (https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/).

In a `pull_request` event from a fork, the token will be read only anyway; this would scope it down to read on the repo contents only; on a PR from a branch on the repo itself, the token would have read/write, and this would scope it down significantly.

Anyway this is completely optional for this PR, just something I've been thinking about.

```suggestion
    name: Lint extra docsite docs and links
    permissions:
      contents: read
    runs-on: ubuntu-latest
```

---

Shouldn't this better be `contents: none` then? This workflow doesn't need any tokens (or does the checkout action needs some?).

---

I  believe your checkout step will require it yeah, I guess it does feel weird since it's a public repo, maybe you're right. Could always try it out with `contents: none` :)

---

Ok, let's use `contents: read` then for now.

---

(At least then this should work out of the box for a private repo.)

---

good point, I didn't even think about private repos; will work out of the box if they pay for github runners, or run their own with that label.